src/styles/card.scss

```diff
@@ -0,0 +1,99 @@
+$darkBlue: #060b35;
+$gray: #616070;
+$mainColor: #00acdc;
+$white: #fff;
```

Comment:
move this to separate file, and don't use name of the color in variable name(avoid using `blue`, `white`...)